internal/pkg/template/templates/workloads/partials/cf/instancerole.yml

```
@@ -38,6 +38,27 @@ InstanceRole:
                 StringEquals:
                   'iam:ResourceTag/copilot-application': !Sub '${AppName}'
                   'iam:ResourceTag/copilot-environment': !Sub '${EnvName}'
+      {{- if hasSecrets .}}
+      - PolicyName: !Join ['', [!Ref AppName, '-', !Ref EnvName, '-', !Ref WorkloadName, SecretsPolicy]]
```

I wonder if we are at risk of hitting the 128 char limit here for the policy name: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html#cfn-iam-policy-policyname

What do you think of simply:
```suggestion
        - PolicyName: 'AccessCopilotTaggedSecrets'
```

---

Ack.